romancal/saturation/saturation_step.py

```diff
@@ -32,6 +31,8 @@ def process(self, input):
                 return result
 
             # Open the reference file data model
+            # Test for reference file
+            self.log.info("Using SATURATION reference file: %s", self.ref_name)
```

Comment:
Is this valid syntax?

Reply:
This works, 
stpipe.ExposurePipeline.saturation - INFO - Using SATURATION reference file: /Users/ddavis/crds_cache/references/roman/wfi/roman_wfi_saturation_0218.asdf

Is there another format that you prefer? This syntax is sprinkled throughout the steps, dark_current, ramp_fit, linearity...

I can convert them to f"" strings if we desire.